apps/calendar/js/templates/multi_day.js

```diff
@@ -0,0 +1,96 @@
+define(function(require, exports, module) {
+'use strict';
+
+var DateSpan = require('./date_span');
+var colorUtils = require('utils/color');
+var create = require('template').create;
+
+module.exports = create({
+
+  hour: function() {
+    var hour = this.arg('hour');
+
+    var html = DateSpan.hour.render({
+      hour: hour,
+      format: this.arg('format'),
+      addAmPmClass: this.arg('addAmPmClass'),
+      className: 'md__display-hour'
+    });
+
+    // we use this temporary element just to grab the textContent
```

Comment:
I don't know how we would do this otherwise.. we need to add the `[aria-label]` and there is no easy way to construct this value unless we grab the whole `textContent`. This is basically the same thing we had before, the only difference is that it wasn't a temporary element, it was reading it from the busytime element itself.